content/correspondence/getting-started/_index.en.md

```diff
@@ -1,14 +1,28 @@
 ---
 title: Getting Started with Altinn Correspondence
 linktitle: Getting Started
-description: Tutorials for how to get started with Altinn 3 Correspondence, for service owners, senders and recipients
+description: Tutorials for how to get started with Altinn 3 Correspondence, for service owners, senders and recipients.
 tags: []
 toc: false
 weight: 30
 ---
 
-{{<children />}}
-
 {{% notice warning  %}}
 Work in progress.
-{{% /notice %}}
\ No newline at end of file
+{{% /notice %}}
+
+{{<children />}}
+
+Due to the different distinct roles an enterprise may have in their interaction with Altinn Correspondence, we have divided our guides based on the specific role;
+
+- Sender
+- Recipient
+- Service Owner
+
+An enterprise may have one, or a combination of all three roles according to the context.
+
+**Service owners** own the configuration and data storage for the Correspondence Service Resource, and will maintain access rights for their specific resource in Altinn Resource Rights Registry.
+
+Most enterprises will have *Sender* and/or *Recipient*, while *Service owners* frequently will have all three.
+
+As a **Service owner**, you are responsible for configuring the service as well as data storage and access management of the service. This means that you as a service owner need to set up and configure the solution before it can be taken into use.
```

Comment:
`configuring the service **as well as data storage** and access management`

I think bold bit can be easy to misinterpret. We are taking responsibility for data storage in general so service owner is only responsible for configuring access management of service.

Reply:
should probably be changed for broker aswell then, its just copied from broker